framework/include/st_things/st_things_types.h

```diff
@@ -15,15 +17,6 @@
  * limitations under the License.
  *
  ******************************************************************/
-/**
- * @ingroup SmartThings
- * @{
- */
-
-/**
- * @file st_things_types.h
- * @brief Provides structure definition for SmartThings Things SDK
- */
```

Comment:
why do you remove?

Reply:
It is from SDKAPI defined by IoT lab. I'll ask.

Reply:
modified